packages/core-api/lib/versions/1/transformers/block.js

```diff
@@ -17,10 +18,10 @@ module.exports = (model) => {
     previousBlock: model.previousBlock,
     height: model.height,
     numberOfTransactions: model.numberOfTransactions,
-    totalAmount: Number(model.totalAmount),
-    totalForged: (Number(model.reward) + Number(model.totalFee)).toString(),
-    totalFee: Number(model.totalFee),
-    reward: Number(model.reward),
+    totalAmount: +bignumify(model.totalAmount).toFixed(),
+    totalForged: +bignumify(model.reward).plus(model.totalFee).toFixed(),
```

Comment:
You replace `.toString()` by `+.....toFixed()` so instead of a string you get a number, is it ok ?

Reply:
Should be fine, we mainly had it as `toString` to prevent any issues.